packages/fxa-settings/src/lib/cache.ts

```diff
@@ -25,6 +27,14 @@ function accounts(accounts?: LocalAccounts) {
 
 export function currentAccount(account?: OldSettingsData) {
   const all = accounts() || {};
+
+  // Current user can be specified in url params (ex. when clicking
+  // `Manage account` from sync prefs.
+  const params = searchParams(window.location.search);
```

Comment:
nit: singular `searchParam('uid', window.location.search)`
tangent: why aren't we using `new URL(window.location).searchParams` in utilities.ts?

Reply:
No reason, I'll update it.